baremaps-core/src/main/java/org/apache/baremaps/storage/geoparquet/GeoParquetDataStore.java

```diff
@@ -30,22 +30,24 @@
 public class GeoParquetDataStore implements DataStore {
 
   private final URI uri;
+  private final String tableName;
 
-  public GeoParquetDataStore(URI uri) {
+  public GeoParquetDataStore(URI uri, String tableName) {
```

Comment:
Not sure we should add the table name here. We can set a new table name in the add(String name, DataTable table) method.